src/App.js

```diff
@@ -3,14 +3,14 @@ import './App.css';
 import Video from './Video/Video'
 import Header from './Component/Header/Header'
 import KikkPro from './Component/KikkPro/KikkPro';
-import Logo from './Logo/Logo';
+import Image1 from './Image1/Image1';
```

Comment:
I wanted you to change the folder name from Logo to images.
You have changed everything. 
Suggested change was

import Logo from './images/Logo';